website/docs/docs/use-dbt-semantic-layer/dbt-semantic-layer.md

```diff
@@ -0,0 +1,133 @@
+
+For more info on Public Preview and plan availability, check out the [Public Preview](/docs/use-dbt-semantic-layer/dbt-semantic-layer#public-preview) and [product architecture](/docs/use-dbt-semantic-layer/dbt-semantic-layer#product-architecture) section.
+    
+:::
+
+The dbt Semantic Layer allows data teams to centrally define essential business metrics like `revenue`, `customer`, and `churn` in the modeling layer (your dbt project) for consistent self-service within downstream data tools like BI and metadata management solutions.
+
+The result? You have less duplicative coding for data teams and more consistency for data consumers. The dbt Semantic Layer has four main parts:
+
+- Define your metrics in version-controlled dbt project code
+- Import your metric definitions via the [Metadata API](/docs/dbt-cloud/dbt-cloud-api/metadata/metadata-querying)
+- Query your metric data via the dbt Proxy Server
+- Explore and analyze dbt metrics in downstream tools
+
+
+<Lightbox src="/img/docs/dbt-cloud/semantic-layer/sl_architecture.png" title="dbt Semantic Layer architecture" />
+
+### What makes the dbt Semantic Layer different?
+
+The dbt Semantic Layer reduces code duplication and inconsistency regarding your business metrics. By moving metric definitions out of the BI layer and into the modeling layer, data teams can feel confident that different business units are working from the same data definitions, regardless of their tool of choice. If a metric definition changes in dbt, it’s refreshed everywhere it’s invoked and creates consistency across all applications.
+
+
+## Prerequisites
+To use the dbt Semantic Layer, you’ll need to meet the following:
+
+
+<VersionBlock firstVersion="1.3" >
+
+<span>&#8226;</span> Have a North American multi-tenant <a href="https://cloud.getdbt.com/">dbt Cloud</a> account. Developer accounts will be able to query the Proxy Server using SQL, but will not be able to browse pre-populated dbt metrics in external tools, which requires access to the Metadata API. <br />
+<span>&#8226;</span> Have both your production and development environments running dbt version 1.3 or higher <br />
+<span>&#8226;</span> Use Snowflake data platform <br />
+<span>&#8226;</span> Install the <a href="https://hub.getdbt.com/dbt-labs/metrics/latest/">dbt metrics package</a> version <code>">=1.3.0", "<1.4.0"</code> in your dbt project <br />
+<span>&#8226;</span> Set up the <a href="https://docs.getdbt.com/docs/dbt-cloud/dbt-cloud-api/metadata/metadata-overview">Metadata API</a> in the integrated tool to import metric definitions <br />
+<span>&#8226;</span> Recommended - Review the <a href="https://docs.getdbt.com/docs/building-a-dbt-project/metrics">dbt metrics page</a> and <a href="https://docs.getdbt.com/blog/getting-started-with-the-dbt-semantic-layer">Getting started with the dbt Semantic Layer</a> blog <br />
```

Comment:
One of the build failures is coming from the `<span>&#8226;</span>` blocks. If intending to use a list here, all `<span>&#8226;</span>` should be replaced with `-`. The `<br />` tags are not needed as each list item will drop to a new line on its own. I would also adjust the links here to use the `[link_name](link_url)` markdown format rather than html, as well as use the ` ` symbols for code blocks rather than `<code>`.

For example, the first bullet in this block would be updated to look like:
```
 - Have a North American multi-tenant [dbt Cloud](https://cloud.getdbt.com/) account. Developer accounts will be able to query the Proxy Server using SQL, but will not be able to browse pre-populated dbt metrics in external tools, which requires access to the Metadata API.
```